pkg/orm/task_result.go

```diff
@@ -45,17 +45,6 @@ func (t *TaskResultORM) GetTaskResultsByTaskId(ctx context.Context, taskId strin
 }
 
 func (t *TaskResultORM) GetCompletedTResultByTaskAndWorker(ctx context.Context, taskId string, workerId string) ([]db.TaskResultModel, error) {
-	cacheKey := cache.BuildCacheKey(cache.TaskResultByTaskAndWorker, taskId, workerId)
-
-	var results []db.TaskResultModel
-	cacheInstance := cache.GetCacheInstance()
-
-	// Try to get from cache
-	if err := cacheInstance.GetCacheValue(cacheKey, &results); err == nil {
-		return results, nil
-	}
-
-	// Cache miss, fetch from database
```

Comment:
also why do we remove this?

Reply:
For this one, task can be submitted multiple times, because the validate function is using the cache, so I guess we shouldn't cache this.